scss/bitstyles/atoms/button--icon-reversed/_.scss

```diff
@@ -14,23 +14,23 @@
     color: var(--button-fg-hover, $bitstyles-button-icon-reversed-color-hover);
     text-decoration: none;
     background: var(--button-bg-hover, $bitstyles-button-icon-reversed-background-color-hover);
-    border: $bitstyles-button-icon-reversed-border-hover;
+    border-color: $bitstyles-button-icon-reversed-border-color-hover;
     outline: none;
     box-shadow: $bitstyles-button-icon-reversed-box-shadow-hover;
   }
 
   &:active {
     color: $bitstyles-button-icon-reversed-color-active;
     background: $bitstyles-button-icon-reversed-background-color-active;
-    border: $bitstyles-button-icon-reversed-border-active;
+    border-color: $bitstyles-button-icon-reversed-border-color-active;
     box-shadow: $bitstyles-button-icon-reversed-box-shadow-active;
   }
 
   &:disabled {
     color: $bitstyles-button-icon-reversed-color-disabled;
     cursor: default;
     background: $bitstyles-button-icon-reversed-background-color-disabled;
-    border: $bitstyles-button-icon-reversed-border-disabled;
+    border-color: $bitstyles-button-icon-reversed-border-color-disabled;
     box-shadow: $bitstyles-button-icon-reversed-box-shadow-disabled;
   }
```

Comment:
So does that mean that we are now adding a border to all the buttons--icons and button--dangers that was not there before?

Reply:
Aha, no that’s from #411 — that got rebased but I didn’t update this branch. I’ve rebased! That PR will need to be merged before this one

(The change here is that border-color can be changed on hover etc, but all buttons will have the same _width_ of border. This means sizing and vertical alignment is equal, and no shifting happens on hover/etc)